proposals/multiple_user_script_worlds.md

```diff
@@ -0,0 +1,285 @@
+# Proposal: Allow multiple user script worlds
+
+**Summary**
+
+Allow developers to configure and use multiple user script worlds in the
+`browser.userScripts` API.
+
+**Document Metadata**
+
+**Author:** rdcronin
+
+**Sponsoring Browser:** Google Chrome
+
+**Contributors:** N/A
+
+**Created:** 2024-03-07
+
+**Related Issues:** TODO
```

Comment:
@Rob--W I didn't see an existing bug for this.  Do you know if one is filed?

Reply:
I think this would be better tracked individually (rather than lumped into the meta "user scripts in MV3" issue), so I've filed a new issue ( issue #565 ) for it, but referenced the original comment.